### ISSUE TITLE
Runtime error on Windows machines for Nuxt 3 (absolute path must be valid)

### ISSUE THREAD

**Issue by Xyrai** [nuxt-modules/robots#99]
Hey there 👋

I am currently experiencing an error on my Windows (11) machine when trying to install the `@nuxtjs/robots 3.0.0` package on `nuxt 3.6.1`.

**Error Log**
```bash
 ERROR  [worker reload] [worker init] Only URLs with a scheme in: file and data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

  at new NodeError (node:internal/errors:399:5)
  at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:1059:11)
  at defaultResolve (node:internal/modules/esm/resolve:1135:3)
  at nextResolve (node:internal/modules/esm/loader:163:28)
  at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
  at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
  at link (node:internal/modules/esm/module_job:76:36)
```

Node version: 18.15.0

**List of things I tried out so far without success:**
1. Using the edge version of Nuxt
https://github.com/nuxt/framework/issues/4232#issuecomment-1345385724

2. Creating a `robots.config.ts` file 
https://github.com/nuxt-modules/robots/issues/86#issuecomment-1366121300

**Here is a repository containing a replica of the error**: https://github.com/Xyrai/nuxt-robots-windows-error/tree/main
Additional link to **[Sandbox](https://codesandbox.io/p/github/Xyrai/nuxt-robots-windows-error/main?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522panelType%2522%253A%2522TABS%2522%252C%2522id%2522%253A%2522cljfcbjd0000b3n6ps6ndhupo%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522panelType%2522%253A%2522TABS%2522%252C%2522id%2522%253A%2522cljfcbjd0000d3n6pw627yuwl%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cljfcbjd0000b3n6ps6ndhupo%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cljfcbjd0000a3n6pwlvc7u7g%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%257D%255D%252C%2522id%2522%253A%2522cljfcbjd0000b3n6ps6ndhupo%2522%252C%2522activeTabId%2522%253A%2522cljfcbjd0000a3n6pwlvc7u7g%2522%257D%252C%2522cljfcbjd0000d3n6pw627yuwl%2522%253A%257B%2522id%2522%253A%2522cljfcbjd0000d3n6pw627yuwl%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cljfcbjd0000c3n6prb3tc351%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522path%2522%253A%2522https%253A%252F%252Fvgs7w7-3000.csb.app%252F%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522cljfcbwzc00gb3n6pqdt10f8n%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%252C%2522activeTabId%2522%253A%2522cljfcbjd0000c3n6prb3tc351%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)**, this runs fine since it is not build on a Windows container.

**Comment by Xyrai** [comment]
Adding a `robots.config.ts` file to the root of your project now works with Nuxt `3.6.1`. Is there any clarification of why adding this file manually fixes the issue, since it is not mentioned in the `README`.

**Comment by jrutila** [comment]
Can confirm this happening with nuxt 3.6.1. The workaround of using `robots.config.ts` works.

**Comment by alexnguyennz** [comment]
Thanks for this workaround - fixes this issue on 3.6.2.

**Comment by TheDutchCoder** [comment]
We've just experienced this exact same issue, and the underlying issue was actually the fact that some of our imports in **TypeScript** required `.js` extensions, otherwise the resolver would create paths like `C:/foo/bar` instead of `file://C:/foo/bar.mjs` in the `.nuxt` output.

I hope this can help others that run into this nearly undebuggable issue.
